public/locales/en/account.json

```diff
@@ -100,8 +100,7 @@
     "edit_card": "Edit card",
     "paypal": "PayPal",
     "update_payment_details": "Update payment details",
-    "update_payment_success": "Your payment details have been updated",
-    "longer_than_usual": "The payment is taking longer than usual. Please try again later."
```

Comment:
I think you've performed an i18n scan, but the removed lines are still in use. We should make sure that they can be found by the scan (either by adding the source type to the i18n config, or adding a code comment).